src/services/homework/index.js

```diff
@@ -21,6 +22,18 @@ module.exports = function() {
 	hwService.before(hooks.before);
 	hwService.after(hooks.after);
 
+	/* Homework model */
+	app.use('/homework/copy', service({
+		Model: homeworkModel,
```

Comment:
Why do you link the homeworkModel here? That would mean, that you can POST /homework/copy to create a homework-entry. That's the same as the homework-service but **without** the security-checks for the school etc.

As @janrenz and @Langleu mentioned before, you should really create an extra copy-service (no model-service) and move the functionality from the hook to the create-method of the service.